modules/ensemble/lib/widget/loading_container.dart

```diff
@@ -53,7 +56,7 @@ class LoadingContainer extends StatefulWidget
       'defaultShimmerPadding': (value) => _controller.padding =
           Utils.getInsets(value), // Backward compatibility
       'shimmerOptions': (options) {
-        if (options is Map<String, dynamic>) {
+        if (options is YamlMap) {
```

Comment:
how will this work when options are being set in javascript? this is something I keep bringing up in every meeting

Reply:
did you test it with js?